packages/now-static-build/test/cache.test.js

```diff
@@ -0,0 +1,26 @@
+const { prepareCache } = require('../dist');
+const path = require('path');
+
+describe('prepareCache', () => {
+  test('should cache yarn.lock, package-lock.json and node_modules', async () => {
+    const files = await prepareCache({
+      workPath: path.resolve(__dirname, './cache-fixtures/default'),
+      entrypoint: 'index.js',
+    });
+
+    expect(files['yarn.lock']).toBeDefined();
+    expect(files['package-lock.json']).toBeDefined();
+    expect(files['node_modules/file']).toBeDefined();
+
+    expect(files['index.js']).toBeUndefined();
+  });
+
+  test('should cache `.cache` folder for gatsby deployments', async () => {
+    const files = await prepareCache({
+      workPath: path.resolve(__dirname, './cache-fixtures/gatsby'),
+      entrypoint: 'package.json',
+    });
+
+    expect(files['.cache/file']).toBeDefined();
```

Comment:
I think you should add `expect(files['package.json']).toBeDefined();` here too, right?

Reply:
It's not cached and I don't think it was cached before either. Should we cache it?

Reply:
Oops, I mean `package-lock.json` or `yarn.lock`

Reply:
Oh, yep. Good idea 👍